dashbuilder-client/dashbuilder-navigation-client/src/main/resources/org/dashbuilder/client/navigation/resources/i18n/NavigationConstants.properties

```diff
@@ -14,20 +14,25 @@
 # limitations under the License.
 #
 
+navigationTitle=Navigation
+treeItem=Tree
+newMenu=New
 newItem=+ New {0}
 gotoItem=Goto {0}
 itemMenuTitle=Show menu
 newItemName=- New {0} -
 editItem=Edit
 deleteItem=Delete
+pageIconTitle=A user created page
+perspectiveIconTitle=A hard-coded static perspective
 moveUp=Move up
 moveDown=Move down
 moveFirst=Move first
 moveLast=Move last
 saveChanges=Save changes?
 save=Save
 cancel=Cancel
-navItemNotFound="Not found: {0}
+navGroupNotFound=The navigation group configured might have been deleted or does not exit. Please, check this component configuration.
```

Comment:
check this component configuration -> check configuration of this component

Reply:
Done